sdk/src/packet.rs

```diff
@@ -29,10 +29,6 @@ bitflags! {
         const REPAIR         = 0b0000_0100;
         const SIMPLE_VOTE_TX = 0b0000_1000;
         const TRACER_PACKET  = 0b0001_0000;
-        /// to be set by bank.feature_set.is_active(round_compute_unit_price::id()) at the moment
-        /// the packet is built.
-        /// This field can be removed when the above feature gate is adopted by mainnet-beta.
-        const ROUND_COMPUTE_UNIT_PRICE = 0b0010_0000;
         /// For tracking performance
         const PERF_TRACK_PACKET  = 0b0100_0000;
```

Comment:
Might be better to change it to `0010_000`, so there would be no "gap"?

Reply:
These are serialized in banking_trace data, so I think best to keep the meaning of each flag consistent.

`ROUND_COMPUTE_UNIT_PRICE` is actually an exception since that was populated only in banking_stage's copy of the `Packet`.

Reply:
Still not great to leave a gap in bit flags. If don't want to introduce a break change in banking_trace at the moment, how about tag `0b0010_0000` as `UNUSED`?

Reply:
69ef158cf1bfdad0f62c70429d5853471aba4900